tools/ci_build/github/windows/setup_env_cuda.bat

```diff
@@ -1 +1 @@
-set PATH=%BUILD_BINARIESDIRECTORY%\packages\python;%BUILD_BINARIESDIRECTORY%\packages\python\DLLs;%BUILD_BINARIESDIRECTORY%\packages\python\Library\bin;%BUILD_BINARIESDIRECTORY%\packages\python\script;C:\local\cudnn-10.0-windows10-x64-v7.3.1.20\cuda\bin;C:\local\cuda_10.0.130_win10;%PATH%;C:\local\systools
+set PATH=%BUILD_BINARIESDIRECTORY%\packages\python;%BUILD_BINARIESDIRECTORY%\packages\python\DLLs;%BUILD_BINARIESDIRECTORY%\packages\python\Library\bin;%BUILD_BINARIESDIRECTORY%\packages\python\script;C:\local\cudnn-10.0-windows10-x64-v7.3.1.20\cuda\bin;C:\local\cuda_10.0.130_win10_trt70011dll;%PATH%;C:\local\systools
```

Comment:
Does trt need a special CUDA?

Reply:
Good catch. Even in the master branch the paths is still set to C:\local\cudnn-10.0-windows10-x64-v7.3.1.20\cuda\bin and C:\local\cuda_10.0.130_win10. They should point to C:\local\cudnn-10.0-windows10-x64-v7.6.5.32\cuda\bin and C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v10.0, right?

Reply:
I think it's better to totally remove them, or duplicate this file, one for windows CUDA, another for TRT build. Because TRT need CUDA 10.2 instead of CUDA 10.0

Reply:
Actually we are using TRT7 win package for CUDA10.0. Is the TRT7 in the Win-GPU-2019 machines for CUDA10.0?

Reply:
we should standardize on one CUDA version for win GPU agent pool. 
@snnn, you were planning on updating GPU agents to CUDA 10.1 right?
if so, that means we should use the TRT7 win package for CUDA 10.1
otherwise, we can continue to use the TRT7 win package for CUDA 10.0

Reply:
Yes, it's under C:\local\TensorRT-7.0.0.11.cuda-10.0.cudnn7.6\TensorRT-7.0.0.11